modules/core/02-client/migrations/v7/solomachine.go

```diff
@@ -50,104 +50,104 @@ func (cs ConsensusState) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error
 }
 
 // ClientType panics!
-func (cs ClientState) ClientType() string {
+func (ClientState) ClientType() string {
 	panic("legacy solo machine is deprecated!")
 }
 
 // GetLatestHeight panics!
-func (cs ClientState) GetLatestHeight() exported.Height {
+func (ClientState) GetLatestHeight() exported.Height {
 	panic("legacy solo machine is deprecated!")
 }
 
 // Status panics!
-func (cs ClientState) Status(_ sdk.Context, _ storetypes.KVStore, _ codec.BinaryCodec) exported.Status {
+func (ClientState) Status(_ sdk.Context, _ storetypes.KVStore, _ codec.BinaryCodec) exported.Status {
 	panic("legacy solo machine is deprecated!")
 }
 
 // Validate panics!
-func (cs ClientState) Validate() error {
+func (ClientState) Validate() error {
 	panic("legacy solo machine is deprecated!")
 }
 
 // ZeroCustomFields panics!
-func (cs ClientState) ZeroCustomFields() exported.ClientState {
+func (ClientState) ZeroCustomFields() exported.ClientState {
 	panic("legacy solo machine is deprecated!")
 }
 
 // Initialize panics!
-func (cs ClientState) Initialize(_ sdk.Context, _ codec.BinaryCodec, _ storetypes.KVStore, consState exported.ConsensusState) error {
+func (ClientState) Initialize(_ sdk.Context, _ codec.BinaryCodec, _ storetypes.KVStore, _ exported.ConsensusState) error {
 	panic("legacy solo machine is deprecated!")
 }
 
 // ExportMetadata panics!
-func (cs ClientState) ExportMetadata(_ storetypes.KVStore) []exported.GenesisMetadata {
+func (ClientState) ExportMetadata(_ storetypes.KVStore) []exported.GenesisMetadata {
 	panic("legacy solo machine is deprecated!")
 }
 
 // CheckForMisbehaviour panics!
-func (cs ClientState) CheckForMisbehaviour(ctx sdk.Context, cdc codec.BinaryCodec, clientStore storetypes.KVStore, msg exported.ClientMessage) bool {
+func (ClientState) CheckForMisbehaviour(ctx sdk.Context, cdc codec.BinaryCodec, clientStore storetypes.KVStore, msg exported.ClientMessage) bool {
```

Comment:
nit: consistency

```suggestion
func (ClientState) CheckForMisbehaviour(_ sdk.Context, _ codec.BinaryCodec, _ storetypes.KVStore, _ exported.ClientMessage) bool {
```

tho I'm seeing most might need to be updated to follow a consistent